src/intTest/java/uk/nhs/digital/nhsconnect/nhais/jms/InboundMeshServiceMultiTransactionTest.java

```diff
@@ -128,14 +132,12 @@ void whenMeshInboundQueueRegistrationMessageIsReceived_thenMessageIsHandled(Soft
         return inboundStateRepository.findBy(WorkflowId.REGISTRATION, SENDER, RECIPIENT, SIS, sms, tn);
     }
 
-    private void assertOutboundQueueRecepMessage(SoftAssertions softly) throws JMSException, IOException {
-        var outboundQueueMessage = getOutboundQueueMessage();
-
-        var meshMessage = parseOutboundMessage(outboundQueueMessage);
+    private void assertOutboundQueueRecepMessage(SoftAssertions softly) throws IOException {
+        List<String> msgs = meshClient.getInboxMessageIds();
+        var meshMessage = meshClient.getEdifactMessage(msgs.get(0));
 
         softly.assertThat(meshMessage.getContent()).isEqualTo(new String(Files.readAllBytes(recep.getFile().toPath())));
         softly.assertThat(meshMessage.getWorkflowId()).isEqualTo(WorkflowId.RECEP);
-        softly.assertThat(meshMessage.getMessageSentTimestamp()).isEqualTo(ISO_RECEP_SEND_TIMESTAMP);
```

Comment:
timestamp is null

Reply:
Why did the timestamp go missing?

Reply:
MeshMessage downloaded from MESH Client doesn't have timestamp - it is added when publishing to Inbound/Outbound queue

Reply:
Method where this takes place should be renamed to not mention queue